openupgrade_scripts/scripts/purchase_requisition/16.0.0.1/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,41 @@
+---Models in module 'purchase_requisition'---
+new model purchase.order.group
+new model purchase.requisition.alternative.warning [transient]
+new model purchase.requisition.create.alternative [transient]
+# NOTHING TO DO
+
+---Fields in module 'purchase_requisition'---
+purchase_requisition / product.template         / purchase_requisition (selection): DEL selection_keys: ['rfq', 'tenders']
```

Comment:
@SonCrits chỗ này cần tìm tất cả purchase order của purchase.requisition (có kiểu Call for tender; purchase_requisition.type_multi để liên kết chúng với nhau
chính là tạo dữ liệu cho model `purchase.order.group`